src/App.css

```diff
@@ -14,14 +14,14 @@
 }
 
 .App-header {
-  background-color: #d6d9dd;
+  background-color: #ffffff;
   min-height: 100vh;
   display: flex;
   flex-direction: column;
   align-items: center;
   justify-content: center;
   font-size: calc(10px + 2vmin);
-  color: white;
+  color: rgb(255, 255, 255);
 }
 
 .App-link {
```

Comment:
If you are not using the `.App-link` className, I suggest removing it.

Reply:
Removed, thanks for the feedback Charlean!